src/components/Navbar.tsx

```diff
@@ -98,8 +99,27 @@ export default function Navbar() {
                         className="sr-only focus:not-sr-only"
                         href="#main-content"
                     >{nav.toContent}</a>
-                    <NavLink class="px-14 lg:px-0" href="/">{nav.home}</NavLink>
-                    <NavLink class="px-14 lg:px-0" href="/förföretag">{nav.forCompanies}</NavLink>
+                     <div className="flex flex-col lg:flex-row items-start h-[100px]">
+                        <div className="mr-[40px] mt-4">
+                            <NavLink class="px-14 lg:px-0" href="/">{nav.home}</NavLink>
+                        </div>
+                        
+                        <div className={`${forCompanyDrop ? 'lg:bg-[#666474] lg:bg-opacity-60 lg:rounded-md lg:w-[200px] w-[420px]' : null} lg:px-4 pb-2`}>
+                            <div className="flex flex-row lg:flex-col mt-4 ">
+                                <div onMouseEnter={() => setForCompanyDrop(true)} onMouseLeave={() => setForCompanyDrop(false)}>
+                                    <NavLink  class="pl-14 pr-4 lg:px-0" href="/förföretag">{nav.forCompanies}</NavLink>
+                                </div>
+                                <div className="lg:block hidden">
+                                {forCompanyDrop ? (
+                                <div onMouseEnter={() => setForCompanyDrop(true)} onMouseLeave={() => setForCompanyDrop(false)}><NavLink class="pr-14 lg:px-0" href="/faq">faq</NavLink></div>) : null }
+                                </div>
+                                {open ? (
+                                <div onMouseEnter={() => setForCompanyDrop(true)} onMouseLeave={() => setForCompanyDrop(false)}><NavLink class="pr-14 lg:px-0" href="/faq">faq</NavLink></div>) : null }
+                                
+                            </div>
+                        </div>
+                        
+                    </div>
```

Comment:
Let's move the redesign of the navbar to another pull request

Reply:
Yes, the pull request is now slitted into two PR.